apps/modernization-ui/src/apps/patient/profile/sexBirth/SexBirth.tsx

```diff
@@ -17,7 +17,10 @@ const asView = (birth?: PatientBirth, gender?: PatientGender): Data[] => [
         title: 'As of:',
         text: internalizeDate(birth?.asOf)
     },
-    { title: 'Current age:', text: birth?.age?.toString() },
+    {
+        title: 'Current age:',
+        text: `${birth?.bornOn ? `${calculateAge(birth?.bornOn)?.quantity?.toString()} ${calculateAge(birth?.bornOn)?.unit}` : ''}`
```

Comment:
suggestion:  Since there is a standard way to display age, this could be encapsulated in an additional `displayAge` function that takes the result of `calculateAge` and outputs the string.  That would also remove the need to define the type with the singular and plural names.  The `displayAge` function would determine that.

Reply:
It looks like { displayAgeAsOfToday } from 'date/displayAge' was what I was trying to accomplish. I used that function instead, with a small tweak to display age as 0 days when the patient date of birth is the same as the current date.